numba/targets/arrayobj.py

```diff
@@ -1739,6 +1783,307 @@ def _increment_indices_array(context, builder, arrty, arr, indices, end_flag=Non
     _increment_indices(context, builder, arrty.ndim, shape, indices, end_flag)
 
 
+def make_nditer_cls(nditerty):
```

Comment:
Hmm, if I read `make_nditer_struct`, I would personally assume it returns a structure instance, not a class, though.